docs/source/model_design/concept_model/results_processing_steps/index.rst

```diff
@@ -24,15 +24,105 @@
+
+- Remember that simulation input data are generally age and sex specific, so it's generally easiest to perform verification of simulation inputs and outputs at the age and sex specific level. However, especially if simulating across many years (which may result in slight changes in the population structure over time), be sure to also **evaluate total population outcome rates as well as age- and sex-specific rates**.
+
+- If something is not verifying, but is close to the target, it could be due to stochastic variation. Check and see if it gets closer when you pool across all scenarios, all ages, etc. If so, stoachstic variation could be to blame for the deviation from the target value. In this case, consider increasing the simulated population size and/or simulated number of draws. Also, try looking at the median value, which may be closer to the target value than the mean is when there are small counts.
+
+General list of things to verify
+-----------------------------------
+
+Generally, we should verify that the simulation outputs match the expected values for everything that is in the simulation artifact, including:
+
+- Population age and sex structure at initialization and over time 
+
+- Risk factor exposures
+
+- Risk factor effects
+
+- All-cause mortality, YLL, YLD, and DALY rates
+  
+  - YLL, YLD, and DALY outcomes are generally not in the artifact, so they will need to be pulled from GBD directly
+
+  - Note that all cause YLD rates are not expected to verify since the simulation outputs will represent YLDs due to modeled causes only. If they differ dramatically, consider adjusting for this in post-processing.
+
+- Cause-specific parameters (including all relevant data such as prevalence, incidence, remission, excess mortality, cause-specific mortality, YLDs, YLLs, DALYs)
+
+- Intervention coverage
+
+- Intervention effects
+
+- Any other modeled parameters!
+
+Verification of these parameters compared to artifact and GBD values is generally done for the baseline scenario and alternative scenarios as necessary (especially for intervention coverage and effect verification). However, they should also be evalutated in alternative scenarios to ensure that there are no unexpected changes.
+
+Additionally, especially if simulating across several years, we should check not only that the parameters meet verification criteria across all simulated years, but also that there are no unacceptable/obvious trends in simulated outputs across simulated years (ex: some mortality rate increasing with time, etc.).
+
```

Comment:
I would add something about when outside data might be needed to validate against. What is an example of a situation when the simulation results are non-trivial and so comparing against the artifact might not make sense? Especially for alternative scenarios, I still struggle with what might be a bug and what might be a result.

Reply:
I just added a new bullet in the latest commit - let me know if you think it addresses your point or not!

Reply:
I think that works yeah!